manifests/f/FastStone/Viewer/7.6/FastStone.Viewer.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: FastStone.Viewer
+PackageVersion: "7.6"
+PackageLocale: en-US
+Publisher: FastStone Soft
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: FastStone Image Viewer
+PackageUrl: https://www.faststone.org/FSViewerDetail.htm
+License: Copyright (c) 2020 FastStone Soft
+LicenseUrl: https://www.faststone.org/order.htm
+# Copyright: 
+# CopyrightUrl: 
+ShortDescription: Fast, stable, user-friendly image browser, converter and editor
+# Description: 
+Moniker: fsviewer
+Tags:
+- admin
+- image
+- viewer
+# Agreements: 
+# ReleaseNotes: 
+# ReleaseNotesUrl: 
+ManifestType: defaultLocale
+ManifestVersion: 1.1.1
```

```suggestion
ManifestVersion: 1.1.0
```
`1.1.1` schema doesn't currently exist.